resources/runners/qemu.sh

```diff
@@ -17,9 +17,14 @@ do
     shift
 done
 
-grep -q -e "vmx" -e "svm" /proc/cpuinfo
-if [ $? -eq 0 ]; then
-  QEMU="$QEMU -enable-kvm"
+# Check for hardware-virtualization support
+if [ "$(egrep -m 1 '^flags.*(vmx|svm)' /proc/cpuinfo)" ]
```

Comment:
Can you check if normal `grep` command works instead of use `egrep`, and if not, change to use equivalent `grep -E`? Also, are the quotes needed? Isn't command expansion just enough?